docs/shared/cypress-plugin.md

```diff
@@ -63,9 +63,7 @@ Cypress doesn't currently re-run your tests after changes are made to applicatio
 
 ### Using Cypress in the Headless Mode
```

Comment:
Can you please change the title to `Using Cypress in the Headed Mode` and modify the text slightly?

```
Cypress runs in headless mode by default (the `headless` option is deprecated). In order to run the Cypress in Headed mode, you can set `browser` option to select the specific browser to be used or set `headed` option to run tests using the default browser.
```

Reply:
Done.